src/router/index.ts

```diff
@@ -9,15 +10,8 @@ const router = createRouter({
       name: 'tiny-mce',
       component: TinyMCE
     },
-    // {
-    //   path: '/about',
```

Comment:
The code looks clean and well-structured. The router is correctly created and exported. Commented out code has been removed, which improves readability. However, the added comment '// Additional routes can be added here' might be unnecessary as it's self-evident.

``` 
// Remove this comment as it's not providing significant value
  ]
```